spec/openapi.yaml

```diff
@@ -1138,6 +1157,40 @@ components:
                 description: Path to package
                 example: "jo/packages"
 
+    Transfer:
+      type: object
+      properties:
+        type:
+          type: string
+          enum: [ NO_TRANSFER, FROM_WEB, FROM_CONNECT, FROM_PRINTER, FROM_SLICER, FROM_CLIENT, TO_CONNECT, TO_CLIENT ]
```

Comment:
I had a comment about this part before on the old API, but there doesn't seem to be a reasonable way to distinguish between „FROM_SLICER“ and „FROM_CLIENT“.

Reply:
Why?

Reply:
Because slicer uses the very same endpoint and protocol as any other client, the only viable way is to guess based on some kind of user agent header. Add to the thing that there isn't just Prusa slicer, but possibly infinite amount of arbitrary slicers out there.

So we have three groups of user agents:

* Browsers the PrusaLink web runs in. This is an unbounded set and not reasonably recognizable.
* Slicers. Again, possibly unbounded and not recognizable set.
* _Something else_. User scripts using curl. User scripts using something to automatically generate STLs, pipe it through slicer and then upload. And is such a script a slicer or not? It's not even clear _for a human_ at that point.